core/jazz_ui/e2e/page-objects/jazzservices.po.ts

```diff
@@ -151,10 +151,10 @@ export class Jazz {
       return element(by.xpath('//div[@class="table-body"]//div[2]//div[contains(text(),"active")]'));
   }
   getMetrices() {
-    return element(by.css('div.tabs-wrap>ul>li:nth-child(4)'));
+    return element(by.xpath('//li[contains(text(),"metrics")]'));
   }
   getTestAPI() {
-    return element(by.xpath('//button[@class="btnT-radial-in"]'));
+    return element(by.xpath('//button[text()="TEST API"]'));
   }
```

Comment:
Wherever available are we using specific id, class, title etc. instead of contains in the xpath search string?